src/renderer/views/Playlist/Playlist.vue

```diff
@@ -107,7 +107,7 @@
               :initial-visible-state="index < 10"
               @move-video-up="moveVideoUp(item.videoId, item.playlistItemId)"
               @move-video-down="moveVideoDown(item.videoId, item.playlistItemId)"
-              @remove-from-playlist="removeVideoFromPlaylist(item.videoId, item.playlistItemId)"
+              @remove-from-playlist="removeVideoFromPlaylist(item.videoId)"
```

Comment:
If I have a playlist with duplicate videos this remove all items with the same video ID
That's not desired